### ISSUE TITLE
Inventory govdelivery - current categories, topics, and stakeholders and make needed decisions

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3061]
### Summary

**What we're after:**
We want to make sure the mailing lists we are using to notify subscribers of website updates are up to date. In working on #1697 I'm finding there's a lot of disorganization, duplication, and bad URLs in our current set up. This task is to fix those issues first before we add the tagged content.

### Related issues

- #1697 

### Completion criteria

- [x] Inventory current topics, categories, notifications, admins and URLs in govdelivery with a new, organized, thorough and current spreadsheet
- [x] Recommend needed updates to stakeholders and get their approval
- [x] Identify pages that govdelivery pagewatch will be enabled. 
- [x] Update spreadsheet as needed
- [x] Update user list for who has access to govdelivery to make changes

### Future work

Once inventory is done and gov delivery topics are set up the way we want them to be, continue with #1697 to add the tags.

**Comment by dorothyyeager** [comment]
The spreadsheet has been made. We've got a lot of follow-up decisions to make - there's some duplicative lists and quite a few lists we're not using. Plus we need to adjust some URLs and add/delete some administrators. I'll chat with @AmyKort on Monday and make a plan for how we'll approach these decisions.

**Comment by dorothyyeager** [comment]
We're going to make recommendations for deleting many of our lists. Where we are keeping a list, we're recommending deleting duplicative lists. As the Information Division uses and maintains the most lists, we're starting with their management and have sent them our recommendations for lists to keep (and needed maintenance) or delete. We'll wait on their response for next steps.

**Comment by dorothyyeager** [comment]
Information Division has gotten back to us with their decisions and we're updating the lists, topics, admins and spreadsheet accordingly. They'll reach out to the other divisions that have lists, so more updates will be coming.

**Comment by dorothyyeager** [comment]
We've made enough progress that as I go through the topics, make the adjustments and update the spreadsheet, it makes sense for me to start adding the tags in #1697 at the same time.

**Comment by dorothyyeager** [comment]
Info has heard back from a couple of the other divisions, so we really are making some progress on this topic. Going to finish the spreadsheet update which will give us the list of admins and pages that have page watch enabled.

**Comment by dorothyyeager** [comment]
The Housekeeping is done. Aside from deciding who should send the email on a couple of shared lists, we've finalized the list of those getting page watch notices and updated URLs for lists that needed them. We've configured error notices. And we deleted 20 duplicative or obsolete lists. We're ready to add the tags now! Closing this issue and picking up with the work in #1697
